docs/helpers.md

```diff
@@ -0,0 +1,75 @@
+# Helper models, views, and ModelAdmins
+
+## OnlineBase
+
+`cms.models.OnlineBase` provides publication controls.
+It provides a single field: `is_online`. When it is `False` (unchecked), it will not appear on the front-end of your website.
```

Comment:
Too much 'it' "When the field is set to `False` (unchecked), the page will not appear on the front-end of your website" or something similar is clearer imo.